utils/api.py

```diff
@@ -459,11 +459,15 @@ def __init__(self, obj):
 
     def reload(self):
         self._obj.reload_if_needed()
+        reloaded_actions = []
         for action in self._obj._actions:
-            setattr(
-                self,
-                action["name"],
-                Action(self, action["name"], action["method"], action["href"]))
+            # don't redefine actions with same name and different methods
```

Comment:
OK, I think that makes sense, perhaps we should put a comment to this effect in here - as I didn't immediately think about different actions being the cause here.